webaudio-audiocontext-close/index.html

```diff
@@ -0,0 +1,140 @@
+<!--
+Copyright 2015 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <meta name="description" content="Sample illustrating the use of WebAudio AudioContext close.">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>WebAudio AudioContext close Sample</title>
+
+    <!-- Add to homescreen for Chrome on Android -->
+    <meta name="mobile-web-app-capable" content="yes">
+    <link rel="icon" sizes="192x192" href="../images/touch/chrome-touch-icon-192x192.png">
+
+    <!-- Add to homescreen for Safari on iOS -->
+    <meta name="apple-mobile-web-app-title" content="WebAudio AudioContext close Sample">
+
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+    <link rel="apple-touch-icon-precomposed" href="../images/apple-touch-icon-precomposed.png">
+
+    <!-- Tile icon for Win8 (144x144 + tile color) -->
+    <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
+    <meta name="msapplication-TileColor" content="#3372DF">
+
+    <link rel="icon" href="../images/favicon.ico">
+
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+
+  <body>
+    <h1>WebAudio AudioContext close Sample</h1>
+
+    <p>Available in <a href="http://www.chromestatus.com/feature/5411681005993984">Chrome 43+</a></p>
+
+    <p>
+      The <code>AudioContext.close</code> method closes the audio context, stopping audio operations, and releases any system resources in use.
+      It will not necessarily release objects <em>created</em> by this <code>AudioContext</code>.
+      This method returns a <code>Promise</code> that resolves when any resources blocking further creation of other audio contexts have been released.
+    </p>
+
+    <div class="output">
+      <div>
+        Manages a pool of AudioContext instances, allowing it to be grown or shrunk (by explicitly closing a random context).
+        If your browser prevents creation of further contexts, close one first.
+      </div>
+      <pre id="count"></pre>
+      <button id="btn-create">Create</button>
+      <button id="btn-close">Close</button>
+      <pre id="results" style="color: red"></pre>
+    </div>
+
+    <!-- // [START code-block] -->
+    <script>
+
+    var pool = [];
+    var i = 0;
```

Comment:
Could you wrap lines 74-86 in an `init()` function or something similar? Might as well take the `i` and `context` variables out of the top-level scope.